File: src/main/java/web/fidex/Application.java
package web.fidex;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.context.annotation.ImportRuntimeHints;

import web.fidex.config.NativeRuntimeHints;

@SpringBootApplication
@EnableCaching
@ImportRuntimeHints(NativeRuntimeHints.class)
public class Application {

	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}

}

File: src/main/kotlin/web/fidex/Application.kt
package web.fidex

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableAsync

@EnableCaching @EnableAsync @SpringBootApplication class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
